runbotics/runbotics-desktop/src/core/websocket/bpmn/runtime-subscriptions.service.ts

```diff
@@ -50,7 +51,13 @@ export class RuntimeSubscriptionsService {
                         event.activity.owner as IActivityOwner
                     ).behaviour;
                     switch (event.activity.content.type) {
-                        case 'bpmn:ServiceTask':
+
+                        case 'bpmn:ErrorEventDefinition': //with BpmnElementType.BOUNDARY_ERROR enum it does not work XD
```

Comment:
wut, nah it has to work. Try to solve it, if there will be no progress then contact me